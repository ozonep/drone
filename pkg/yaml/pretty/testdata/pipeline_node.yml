kind: pipeline
name: default

steps:
  - name: build
    image: golang
    commands:
      - go build

  - name: test
    image: golang
    commands:
      - go build
      - go test -v
    depends_on:
    - build

node:
  disk: ssd
