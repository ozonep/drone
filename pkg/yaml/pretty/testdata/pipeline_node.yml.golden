---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: build
  image: golang
  commands:
  - go build

- name: test
  image: golang
  commands:
  - go build
  - go test -v
  depends_on:
  - build

node:
  disk: ssd

...
